compiler/bindings/python/iree/compiler/tools/import_onnx/__main__.py

```diff
@@ -71,6 +274,11 @@ def load_onnx_model(args: argparse.Namespace) -> onnx.ModelProto:
         raw_model = onnx.load(args.input_file, load_external_data=False)
         onnx.load_external_data_for_model(raw_model, str(args.data_dir))
 
+    if args.opset_version:
+        raw_model = onnx.version_converter.convert_version(
```

Comment:
We should add a check that the provided opset version is indeed greater than the existing one. You can access the opset version from the model via `raw_model.opset_import[0].version`